src/index.html

```diff
@@ -22,6 +22,388 @@
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo of company MOYO"
+          class="header__logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="is-active nav__link"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptopsandcomputers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="image of IMAC"
+          class="card__picture-imac"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <h2 class="card__product-code-imac">Product code: 195434</h2>
+
+        <div class="card__rewiev">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__text-amount-review">Reviews: 5</p>
+        </div>
+
+        <div class="card__price-product">
+          <p class="card__text-price">Price:</p>
+          <p class="card__product-price">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          data-qa="card-hover"
+        >
+          <button class="card__button">Buy</button>
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```